en/getting-started/first-game/index.md

```diff
@@ -588,359 +1081,46 @@ The game character is just running forward, with no purpose. Adding game rules t
+
     ```ts
     init() {
         // ...
         this.playerCtrl.reset();
     }
     ```
 
-## Step counting display
+## Step Count Display
 
-We can display the current number of steps jumped in the interface. Perhaps watching the continuous growth of steps during the jump will be very fulfilling to the player.
+We can display the current number of steps on the interface so that it will be a great sense of accomplishment to watch the number of steps growing during the jump.
 
-1. Create a new label named `Steps` under __Canvas__, adjust the *position*, *font size* and *other properties*.
+1. Create a new Label node named Steps under Canvas and adjust the position, font size and other properties.
 
     ![steps label](./images/steps-label.png)
 
-2. Reference the `Steps` label in `GameManager`
+2. Reference this Label in the `GameManager` script:
 
     ```ts
     @property({type: Label})
-    public stepsLabel: Label|null = null;
+    public stepsLabel: Label | null = null;
     ```
 
-    ![steps label to game manager](./images/add-steps-to-game-manager.png)
+   After saving the script and returning to the editor, drag the Steps node into the GameManager's stepsLabel property box in the properties inspector at
 
-3. Update the current `Step` data to appear in new `Steps` Label. A game ending interface has yet to be created, for now, reset the number of steps to __0__ when restarting playing.
+   ![steps label to game manager](./images/add-steps-to-game-manager.png)
 
-    ```ts
-    set curState (value: GameState) {
-        switch(value) {
-            case GameState.GS_INIT:
-                this.init();
-                break;
-            case GameState.GS_PLAYING:
-                if (this.startMenu) {
-                    this.startMenu.active = false;
-                }
-                if (this.stepsLabel) {
-                    //  reset the number of steps to 0
-                    this.stepsLabel.string = '0';
-                }
-                // set active directly to start listening for mouse events directly
-                setTimeout(() => {
-                    if (this.playerCtrl) {
-                        this.playerCtrl.setInputActive(true);
-                    }
-                }, 0.1);
-                break;
-            case GameState.GS_END:
-                break;
-        }
-        this._curState = value;
-    }
-    ```
-
-    Update the `Steps` Label in a function that responds to the character jumping. It should make sense that recording the number of `Steps` would take place after each and every jump for accuracy.
+3. Update the current step count data to the Steps node. Since we don't have an end screen now, we will jump back to the start screen when the game is over, so we need to see the last jump in the start screen, so we need to reset the step count to 0 when we enter the Playing state.
```

Comment:
is it better to use the words below:
end screen -> GameOver UI?
start screen -> start menu ?